index.html

```diff
@@ -93,11 +115,9 @@ <h2>Profesional Art <br />Printing Data</h2>
                         <a href="#">RUBY</a>
                     </li>
                 </ul>
-                <button type="button">See project</button>
+
             </article>
-        </section>
 
-        <section class="center-content">
             <article class="content-desc">
```

Comment:
- [ ] For the rest of the projects after the first main one. The feature of each of the project is that when you move your mouse over each of them, the project card changes to the first project and the mouse is moved away from the project card, the project card changes to the second project. At this moment, the hover effect is not working on your project card. Kindly look into that and fix.

### First project
![frist-project](https://user-images.githubusercontent.com/26597432/236642081-0cd65ff4-f2f4-4f54-add6-be3ae340cfa6.png)

### Second project
![second-project](https://user-images.githubusercontent.com/26597432/236642091-f843b788-cf58-472e-ad93-ba39e7a603a6.png)

- [ ] You need to increase the height of your project card so as to match the Figma design

### Your Design
![your-rest-projects](https://user-images.githubusercontent.com/26597432/236642172-e1a20d1b-9be0-4bed-93a8-4ef43e2012f0.png)

### Figma Design
![temo-rest-projects](https://user-images.githubusercontent.com/26597432/236642180-9e2c3863-a00b-4d5f-b2bd-cca833dca58c.png)